transforms/universal/pdf2md/python/README.md

```diff
@@ -0,0 +1,60 @@
+# Ingest PDF to Parquet
+
+This tranforms iterate through PDF files or zip of PDF files and generates parquet files
+containing the converted document in Markdown format.
+
+The PDF conversion is using the [Docling package](https://github.com/DS4SD/docling).
+
+
+## Output format
+
+The output format will contain all the columns of the metadata CSV file,
+with the addition of the following columns
+
+```jsonc
+{
+    // all the fields of the input CSV + ...
+
+    "source_filename": "string",  // the basename of the source archive or file
+    "filename": "string",         // the basename of the PDF file
+    "contents": "string",         // the content of the PDF
+    "document_id": "string",      // the document id, corresponding to the hash of the content 
+    "ext": "string",              // the detected file extension
+    "hash": "string",             // the hash of the `contents` column
+    "size": "string",             // the size of `contents`
+    "date_acquired": "date",      // the date when the transform was executing
+    "num_pages": "number",        // number of pages in the PDF
+    "num_tables": "number",       // number of tables in the PDF
+    "num_doc_elements": "number", // number of document elements in the PDF
+    "pdf_convert_time": "float",  // time taken to convert the document in seconds
+}
+```
+
+
+## Parameters
+
+| Parameter  | Default  | Description  |
+|------------|----------|--------------|
+| `--pdf2md_modelsdir`                  | `./artifacts` | The location where the models are located or downloaded to |
```

Comment:
We want to support configuraiton of the transform independent of the CLI.  Can you discuss/doc the configuration keys that the transform's initializer() supports.  In general, these keys should not include the CLI prefix (pdf2md_) since the prefix is used distinguish the args from other cli args, but not needed when configuring the transform directly.

Reply:
These are input parameters, so they are fine

Reply:
So should we remove `--pdf2md_` from the README? the usage and meaning is still the same.
from the latest commit, the transform doesn't require the prefix anymore for its config

Reply:
this is done